samples/apps/copilot-chat-app/webapp/src/libs/connectors/useConnectors.ts

```diff
@@ -61,21 +41,72 @@ export const useConnectors = () => {
      * using MS Graph API token
      */
     const invokeSkillWithGraphToken = async (ask: IAsk, skillName: string, functionName: string) => {
-        return await invokeSkillWithConnectorToken(ask, skillName, functionName, Constants.msGraphScopes);
+        return await invokeSkillWithMsalToken(
+            ask,
+            skillName,
+            functionName,
+            Constants.msGraphScopes,
+            AuthHeaderTags.MsGraph,
+        );
     };
 
     /**
      * Helper function to invoke SK skills
-     * using ADO token
+     * with GitHub token.
+     */
+    const invokeSkillWithGitHubToken = async () => {
+        // TODO: For testing, change as needed
```

Comment:
Remove TODO and rephrase it as an example "This is an example of invoking openAPi skills, follow this format:
...
describe the input and variable params and what the map to on the kernel
...
"